docker/cygnus-ngsi/Dockerfile

```diff
@@ -36,77 +36,95 @@ ENV CYGNUS_MYSQL_PASS "mysql"
 ENV CYGNUS_MONGO_USER "mongo"
 ENV CYGNUS_MONGO_PASS "mongo"
 
+# NOTE: Configure correctly GIT_URL_CYGNUS and GIT_REV_CYGNUS for each git branch/fork used
 ENV GIT_URL_CYGNUS "https://github.com/telefonicaid/fiware-cygnus.git"
+# ENV GIT_URL_CYGNUS "https://github.com/myuser/fiware-cygnus.git"
 ENV GIT_REV_CYGNUS "master"
+# ENV GIT_REV_CYGNUS "mybranch"
 
-ENV MVN_VER "3.2.5"
-ENV MVN_TGZ "apache-maven-${MVN_VER}-bin.tar.gz"
-ENV MVN_URL "http://www.eu.apache.org/dist/maven/maven-3/${MVN_VER}/binaries/${MVN_TGZ}"
+ENV MVN_VER "3.3.9"
 ENV MVN_HOME "/opt/apache-maven"
 
 ENV FLUME_VER "1.4.0"
-ENV FLUME_TGZ "apache-flume-${FLUME_VER}-bin.tar.gz"
-ENV FLUME_URL "http://archive.apache.org/dist/flume/${FLUME_VER}/${FLUME_TGZ}"
 ENV FLUME_HOME "/opt/apache-flume"
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.6.0
+ENV JAVA_VERSION=1.7.0
 
 # Install
 RUN \
     # Add Cygnus user
     adduser ${CYGNUS_USER} && \
-    yum -y install nc java-1.6.0-openjdk-devel git && \
+    yum -y install nc java-${JAVA_VERSION}-openjdk-devel git && \
+    export JAVA_HOME=/usr/lib/jvm/java-${JAVA_VERSION} && \
+    export MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8 -Dproject.build.sourceEncoding=UTF-8 -Dmaven.compiler.useIncrementalCompilation=false -DdependencyLocationsEnabled=false -XX:+TieredCompilation -XX:TieredStopAtLevel=1" && \
```

Comment:
As discussed offline, I think this wont affect the JVM but only Maven.

`-Dfile.encoding=UTF-8` must be added to the starting command within `cygnus-entrypoint.sh` instead.